src/wp-admin/includes/upgrade.php

```diff
@@ -2202,6 +2200,19 @@ function upgrade_550() {
 	}
 }
 
+/**
+ * Executes changes made in WordPress 5.6.0.
+ *
+ * @ignore
+ * @since 5.6.0
+ */
+function upgrade_560() {
+	// Prevent an extra database query on each page-load if the WPLANG option doesn't exist.
+	if ( ! is_multisite() && false === get_option( 'WPLANG' ) ) {
+		update_option( 'WPLANG', '' );
+	}
+}
```

Comment:
This function is actually not required because `populate_options()` handles this already for us once `$wp_db_version` gets updated. It usually holds the commit number.